db/seeds.rb

```diff
@@ -16,16 +16,20 @@
 first_city_id = City.first.id
 
 # seed route
-Route.create(start_id: first_city_id + 1, destination_id: first_city_id + 2, departure_datetime: "2017-10-10 10:00:00", arrival_datetime: "2017-10-10 15:00:00", price_coefficient: 1.5)
-Route.create(start_id: first_city_id + 2, destination_id: first_city_id + 1, departure_datetime: "2017-10-10 10:00:00", arrival_datetime: "2017-10-10 15:00:00", price_coefficient: 2)
-Route.create(start_id: first_city_id + 1, destination_id: first_city_id + 3, departure_datetime: "2017-10-12 10:00:00", arrival_datetime: "2017-10-12 15:00:00", price_coefficient: 1.5)
-Route.create(start_id: first_city_id + 1, destination_id: first_city_id + 4, departure_datetime: "2017-10-12 10:00:00", arrival_datetime: "2017-10-12 15:00:00", price_coefficient: 3)
-Route.create(start_id: first_city_id + 4, destination_id: first_city_id + 2, departure_datetime: "2017-10-15 10:00:00", arrival_datetime: "2017-10-15 15:00:00", price_coefficient: 1.1)
-Route.create(start_id: first_city_id + 3, destination_id: first_city_id + 2, departure_datetime: "2017-10-15 10:00:00", arrival_datetime: "2017-10-15 15:00:00", price_coefficient: 1.3)
-Route.create(start_id: first_city_id + 4, destination_id: first_city_id + 5, departure_datetime: "2017-10-17 10:00:00", arrival_datetime: "2017-10-17 15:00:00", price_coefficient: 2.5)
-Route.create(start_id: first_city_id + 5, destination_id: first_city_id + 7, departure_datetime: "2017-10-17 10:00:00", arrival_datetime: "2017-10-17 15:00:00", price_coefficient: 2)
-Route.create(start_id: first_city_id + 7, destination_id: first_city_id + 3, departure_datetime: "2017-10-20 10:00:00", arrival_datetime: "2017-10-20 15:00:00", price_coefficient: 1)
-Route.create(start_id: first_city_id + 7, destination_id: first_city_id + 2, departure_datetime: "2017-10-20 10:00:00", arrival_datetime: "2017-10-20 15:00:00", price_coefficient: 1.2)
+(1..5).each do |daystep|
+  date = DateTime.new(2017, 10, 10+daystep, 10, 0 ,0);
+  (0..City.count).each do |i|
+    (0..City.count).each do |j|
+      if i != j
+        Route.create(start_id: first_city_id + i,
+                     destination_id: first_city_id + j,
+                     departure_datetime: date.to_s,
```

Comment:
It doesn't make sense to convert datetime to string

Reply:
Oh, forgot. Also use namings like `started_at/departured_at` instead specification of field type.